guidance_plugin_validator/src/guidance_plugin_validator/guidance_plugin_components.py

```diff
@@ -0,0 +1,402 @@
+#!/usr/bin/env python
+
+"""
+ * Copyright (C) 2021 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+"""
+import rospy
+
+class StrategicPluginResults:
+    """
+    Convenience class used to store the validation results (both required and optional) 
+    for a Guidance Strategic Plugin. 
+    """
+
+    def __init__(self, plugin_name):
+        """Constructor for StrategicPluginResults"""
+
+        # Set the plugin's name, as used by the plugin_discovery topics and the plugin's advertised service(s)
+        self.plugin_name = plugin_name
+        
+        # Set the plugin's node name
+        if plugin_name == "RouteFollowing":
```

Comment:
Can you explain a bit more about why this special case for RFP is required?

Reply:
Or does this currently only support validating RFP?

Reply:
If we simply need the node name based on the plugin name it should be "/guidance/plugins/<PLUGIN_NAME>" by design, though I'm unsure if our current implementation is correct for that.

Reply:
I saw that you created a new Github Issue #1400 to address this. This issue exists for all current strategic, tactical, and control plugins from what I have seen so far. Once it is addressed, I can update this casing that exists in the Guidance Plugin Validator.